packages/pinia/__tests__/storeSetup.spec.ts

```diff
@@ -131,4 +131,34 @@ describe('store with setup syntax', () => {
     expect(store.counter).toBe(2)
     expect(counter.value).toBe(2)
   })
+
+  it('handles writeable computed refs', () => {
+    const counter = ref(1)
+    const double = computed({
+      get: () => counter.value * 2,
+      set(val) {
+        counter.value = val / 2
+      },
+    })
+    const useStore = defineStore('main', () => ({
+      counter,
+      double,
+    }))
+    const store = useStore()
+    expect(store.$state.counter).toBe(1)
+    expect(store.$state.double).toBe(2)
```

Comment:
Note a computed property **cannot appear** in the state as the source of truth is somewhere else.